config/controller.yaml

```diff
@@ -114,8 +114,8 @@ spec:
               key: image
         resources:
           requests:
-            cpu: 10m
-            memory: 50Mi
+            cpu: 20m
```

Comment:
Is `20m` consistent with your observations? I didn't pay much attention to cpu.

Reply:
yup. only when the controller is doing things though.